dsl_parser/constraints.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import absolute_import
```

Comment:
Do you really need to import absolute_import, or can you just make the imports actually absolute?

Reply:
yeah good point, this isnt needed, removed

the explicit-relative imports would work just fine without this `__future__`, but i've made them absolute anyway as well